sdk/template/template/samples/v1/typescript/src/eventhubs-checkpointstore-blob/src/blobCheckpointStore.ts

```diff
@@ -0,0 +1,366 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I'm not sure how this file got here, but we shouldn't be adding anything to `sdk/template/template`.